packages/dynamics-lib/src/entities/__tests__/permission.entity.spec.js

```diff
@@ -6,37 +6,53 @@ describe('permission entity', () => {
     optionSetData = await retrieveGlobalOptionSets().cached()
   })
 
-  it('maps from dynamics', async () => {
-    const permission = Permission.fromResponse(
-      {
-        '@odata.etag': 'W/"186695153"',
-        defra_permissionid: '347a9083-361e-ea11-a810-000d3a25c5d6',
-        defra_name: '00000000-2WC3FDR-CD379B',
-        defra_issuedate: '2019-12-13T09:00:00Z',
-        defra_startdate: '2019-12-14T00:00:00Z',
-        defra_enddate: '2020-12-13T23:59:59Z',
-        defra_stagingid: '71ad9a25-2a03-406b-a0e3-f4ff37799374',
-        defra_datasource: 910400003,
-        defra_licenceforyou: 1
-      },
-      optionSetData
-    )
+  describe('maps from dynamics', () => {
+    const getPermission = () =>
+      Permission.fromResponse(
+        {
+          '@odata.etag': 'W/"186695153"',
+          defra_permissionid: '347a9083-361e-ea11-a810-000d3a25c5d6',
+          defra_name: '00000000-2WC3FDR-CD379B',
+          defra_issuedate: '2019-12-13T09:00:00Z',
+          defra_startdate: '2019-12-14T00:00:00Z',
+          defra_enddate: '2020-12-13T23:59:59Z',
+          defra_stagingid: '71ad9a25-2a03-406b-a0e3-f4ff37799374',
+          defra_datasource: 910400003,
+          defra_licenceforyou: 1,
+          defra_ismultibuy: true
+        },
+        optionSetData
+      )
 
-    const expectedFields = {
+    const getExpectedFields = () => ({
       id: '347a9083-361e-ea11-a810-000d3a25c5d6',
       referenceNumber: '00000000-2WC3FDR-CD379B',
       issueDate: '2019-12-13T09:00:00Z',
       startDate: '2019-12-14T00:00:00Z',
       endDate: '2020-12-13T23:59:59Z',
       stagingId: '71ad9a25-2a03-406b-a0e3-f4ff37799374',
       dataSource: expect.objectContaining({ id: 910400003, label: 'Web Sales', description: 'Web Sales' }),
-      isLicenceForYou: expect.objectContaining({ id: 1, label: 'Yes', description: 'Yes' })
-    }
+      isLicenceForYou: expect.objectContaining({ id: 1, label: 'Yes', description: 'Yes' }),
+      isMultiBuy: true
```

Comment:
Looks good, only comment being do we want to check that state is not set when isn't a multi buy?